core/src/test/java/org/apache/calcite/sql/parser/parserextensiontesting/ExtensionSqlParserTest.java

```diff
@@ -32,7 +32,8 @@
 public class ExtensionSqlParserTest extends SqlParserTest {
 
   @Override protected SqlParserImplFactory parserImplFactory() {
-    return ExtensionSqlParserImpl.FACTORY;
+    // return ExtensionSqlParserImpl.FACTORY;
+    return null;
```

Comment:
Why we return a null factory ?

Reply:
@danny0405 thanks for the comment, you are absoluetly right. I comment that usually out (as it has sometimes compilation issues with maven) but should not have commited that. Is fixed now.